### PR TITLE
Lokalise latest may 14

### PR DESCRIPTION
**Pull request by alpita-masurkar**
#### Description:
The latest pull from Lokalise, May 14
Also fixed a typo and refactored subcription to subscription

### CODE REVIEW

File: app/views/ChooseProvider.js [hunk 1]
```diff
@@ -485,7 +485,7 @@ const styles = StyleSheet.create({
   textInput: {
     marginLeft: 10,
   },
-  autoSubcribe: {
+  autoSubscribe: {
```
Comment (tstirrat) [inline app/views/ChooseProvider.js:488]: ideally don't include code changes in language changes. But I'll allow it for this time

File: app/services/HCAService.js [hunk 2]
```diff
@@ -101,8 +101,8 @@ class HCAService {
    */
   async pushAlertNewSubscribedAuthorities(count) {
     await PushNotification.localNotification({
-      title: languages.t('label.authorities_new_subcription_title', { count }),
-      message: languages.t('label.authorities_new_subcription_msg', { count }),
+      title: languages.t('label.authorities_new_subscription_title', { count }),
```
Comment (tstirrat) [inline app/services/HCAService.js:104]: make code changes in a separate PR please.

Comment (tstirrat) [reply app/services/HCAService.js:104]: The way it will work is. change only the en.js and the code in a separate PR. this will sync up to lokalise and will rename and should copy all the translations

Comment (alpita-masurkar) [reply app/services/HCAService.js:104]: ok. will do

File: app/locales/en.json [hunk 3]
```diff
@@ -120,12 +120,15 @@
...
     "news_title": "Latest news",
     "no_data": "No data",
+    "privacy_policy": "Privacy Policy",
     "push_at_risk_message": "You have crossed paths with a COVID-19 patient",
     "push_at_risk_title": "You may be at risk",
     "see_exposure_history": "See exposure history",
     "settings_title": "Dashboard",
     "skip_this_step": "Skip this step",
-    "privacy_policy": "Privacy Policy"
+    "team": "Team",
```
Comment (tstirrat) [inline app/locales/en.json:129]: oh dammit. @alpita-masurkar can you make another PR to remove team, team_para and terms_of_use from en.json.. they are not in the app anymore
